application-analytics-ui/src/main/resources/Analytics/Code/AnalyticsTranslations.xml

```diff
@@ -46,16 +46,36 @@ Analytics.Code.AnalyticsDashboardClass_endDate.hint=End time of the date interva
 Analytics.Code.AnalyticsDashboardClass_description.hint=Additional information to be displayed.
 Analytics.Code.AnalyticsDashboardClass_DisplayedStatistics.hint=List of statistics widgets that will be displayed in the current dashboard, in order.
 
+#Config
+Analytics.Code.ConfigurationClass.matomoTrackingCode.hint=Tracking code received from Matomo. The 'u' variable might be different than the one that is initially generated by Matomo if the server you want to track is on another machine. To fix this, go to the integration tab, copy the Matomo URL, and replace the value of 'u' with the Matomo URL.
+Analytics.Code.ConfigurationClass.idSite.hint=The ID of the site that you want to see the statistics for (found in the integration tab).
+Analytics.Code.ConfigurationClass.requestAddress.hint=The Matomo url (found in the integration tab).
+Analytics.Code.ConfigurationClass.authToken.hint=Token generated by Matomo to be capable of using the API to view the data.
+
+#Most Viewed Pages 
```

Comment:
Use `##` to mark a line as a comment. Also, a space will be necessary, as a best practice. See https://github.com/xwikisas/xwiki-pro-macros/blob/main/xwiki-pro-macros-ui/src/main/resources/XWiki/Macros/Translations.xml#L57